main/src/ca/uwaterloo/flix/language/phase/Verifier.scala

```diff
@@ -235,6 +242,68 @@ object Verifier {
           check(expected = MonoType.Enum(sym.enumSym))(actual = t1, loc)
           tpe
 
+        case AtomicOp.ArrayLength =>
+          val List(arrt) = ts
```

Comment:
the variable name makes sense, but for consistency's sake, these variables should just be called t or t1 t2 t3 ...

This goes for all  the cases